opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/BAJsonFormUtilsTest.java

```diff
@@ -97,6 +107,45 @@ public void getAutoJsonEditMemberFormStringReturnsCorrectJSONObject() throws JSO
         Assert.assertEquals(formTitle, stepOne.getString(TITLE));
     }
 
+    @Test
+    public void getStartFormActivityReturnsCorrectIntent() {
+        Context context = RuntimeEnvironment.application;
+        Intent testIntent = FormUtils.getStartFormActivity(new JSONObject(), "test form", context);
+        Assert.assertNotNull(testIntent);
+        Form form = (Form) Objects.requireNonNull(testIntent.getExtras()).get(JsonFormConstants.JSON_FORM_KEY.FORM);
+        assertEquals("test form", form.getName());
+    }
+
+    @Test
+    public void testComputeDOBUnknown() {
+        JSONObject jsonObj = new JSONObject();
+        JSONObject optionsObject = new JSONObject();
+        try {
+            jsonObj.put(org.smartregister.family.util.JsonFormUtils.READ_ONLY, false);
+            optionsObject = jsonObj.getJSONArray(org.smartregister.family.util.Constants.JSON_FORM_KEY.OPTIONS).getJSONObject(0);
+            optionsObject.put(org.smartregister.family.util.JsonFormUtils.VALUE, Utils.getValue(client.getColumnmaps(), org.smartregister.family.util.Constants.JSON_FORM_KEY.DOB_UNKNOWN, false));
+        } catch (JSONException e) {
+            e.printStackTrace();
+        }
+        Assert.assertNotNull(optionsObject);
+    }
+
+    @Test
+    public void testComputeSurname() {
+        JSONObject jsonObj = new JSONObject();
+        if (clientEvent != null) {
+            try {
+                jsonObj.put(org.smartregister.family.util.JsonFormUtils.VALUE,
+                        (clientEvent.getLastName() == null ? Mockito.anyString() : clientEvent.getLastName()));
+                jsonObj.put(org.smartregister.family.util.JsonFormUtils.READ_ONLY, true);
+            }catch (Exception e){
+                e.printStackTrace();
+            }
+        }
+
+        Assert.assertNotNull(jsonObj);
+    }
```

Comment:
Is this test necessary? The Json Object will never be null.

Reply:
Removed